### PR TITLE
Mixed Audio Only : apiview review comments

### PR DESCRIPTION
**Pull request by ninikasharma**

### CODE REVIEW

File: sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallAsyncLiveTests.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public void runAllClientFunctionsWithResponseAsync(HttpClient httpClient) {
             serverCallAsync = callingServerAsyncClient.initializeServerCall(groupId);
 
             Response<StartCallRecordingResult> startRecordingResponse =
-                serverCallAsync.startRecordingWithResponse(CALLBACK_URI).block();
+                serverCallAsync.startRecordingWithResponse(CALLBACK_URI, null, null, null, null).block();
```
Comment (cochi2) [inline sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallAsyncLiveTests.java:106]: are you adding tests where the null values are actual values?

File: sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallUnitTests.java [hunk 2]
```diff
@@ -53,10 +53,9 @@ public void startRecordingWithRecordingParamsRelativeUriFails() {
 
         assertThrows(
             InvalidParameterException.class,
-            () -> serverCall.startRecording("/not/absolute/uri", RecordingChannelType.MIXED, RecordingContentType.AUDIO_VIDEO, RecordingFormatType.MP4));
+            () -> serverCall.startRecordingWithResponse("/not/absolute/uri", RecordingChannelType.MIXED, RecordingContentType.AUDIO_VIDEO, RecordingFormatType.MP4, null));
```
[the remark this answers is not in the document]
Comment (ninikasharma) [reply sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallUnitTests.java]: Its is being testing in a different test case already :)

File: sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallAsyncLiveTests.java (hunk 1 above)
Comment (ninikasharma) [reply sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallAsyncLiveTests.java]: yes, thats added :)

**Review by chrwhit — APPROVED** [review]
